docs/source/indepth_tutorial/directed-search.ipynb

```diff
@@ -98,9 +98,9 @@
      "name": "stderr",
```

Comment:
<p>I would mention the 5 metrics used ("generational_distance", "hypervolume", "epsilon_indicator", "inverted_gd" and "spacing"), and briefly explain what each one does. That makes discussion later on much more valuable.</p><p> </p><p>A link to an overview of all 5, or a individual link to each one would also suffice.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quaquel/EMAworkbench/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>the provided links contain the five metrics so the information is already there</p>

Reply:
<p>Then maybe add which metrics are described in which papers</p>